plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -0,0 +1,862 @@
+             to this function 'get_access_point_configuration' function
+            in the 'wireless' family to get AP configuration data.
+        """
+        ap_response = None
+        input_param = {}
+        input_param["key"] = ap_ethernet_mac_address
+        try:
+            ap_response = self.dnac._exec(
+                family="wireless",
+                function='get_access_point_configuration',
+                params=input_param,
+            )
+        except Exception as e:
+            self.log("Unable to get the Accesspoint configuratoin for '{0}' ."\
+                        .format(str(input_param) + str(e)), "WARNING")
+
+        if ap_response:
+            self.keymap = self.keymaping(self.keymap, ap_response)
+            ap_response = self.camel_to_snake_case(ap_response)
+            current_configuration = ap_response
+            self.log("Received API response from 'get_access_point_configuration': {0}"\
+                        .format(str(current_configuration)), "DEBUG")
+            ap_selected_field = self.payload.get("config")[0].get("ap_selected_field")
+            if ap_selected_field is None or ap_selected_field == "" or ap_selected_field == "all":
+                self.payload["device_config"] = ap_response
+            else:
+                self.payload["device_config"]=self.data_frame(ap_selected_field,[ap_response])
+
+            self.log("AP configuration {0} exists in Cisco Catalyst Center"\
+                        .format(str(self.payload["device_config"][0])), "INFO")
+            accesspoint_exists = True
+        return (accesspoint_exists, current_configuration)
+
+
+    def compare_ap_cofig_with_inputdata(self, apconfig):
+        """
+        This function used to compare with the input ap detail with the current ap configuration
+        information are not same, those data will be updated in the AP input information.
+        Parameters:
+          - apconfig: This is response of the get_have
+        Returns:
+            This will be the return the final data for update AP detail.
+            "final_input": [
+                {
+                    "adminStatus": true,
+                    "apList": [
+                        {
+                            "macAddress": "34:5d:a8:0e:20:b4"
+                        }
+                    ],
+                    "configureLedBrightnessLevel": true,
+                    "ledBrightnessLevel": 4,
+                    "macAddress": "34:5d:a8:0e:20:b4"
+                } ]
+        Example:
+            functions = Accesspoint(module)
+            final_input_data = functions.compare_ap_cofig_with_inputdata(all_apconfig)
+        """
+
+        newdict = {}
+        if self.want and apconfig:
+            if self.want.get("mac_address") == apconfig["mac_address"] or \
+                    self.want.get("hostname") == apconfig["ap_name"]:
+                allkey = list(self.want.keys())
```

Comment:
Please kindly follow unique style:
use all_key instead of allkey Or we can say configurable_keys 


```
            excluded_keys = {"mac_address", "hostname", "management_ip_address", "family"}
            for value in excluded_key:
```